dart/lib/src/utils/breadcrumb_log_level.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:meta/meta.dart';
+
+import '../../sentry.dart';
+
+/// Determine a breadcrumb's log level (only `warning` or `error`) based on an HTTP status code.
+@internal
+SentryLevel? getBreadcrumbLogLevelFromHttpStatusCode(int statusCode) {
+  // NOTE: null defaults to 'info' in Sentry
+  if (statusCode >= 400 && statusCode < 500) {
+    return SentryLevel.warning;
+  } else if (statusCode >= 500) {
```

Comment:
`m`: I would also add the upper bound for 5xx. There are weird non standard HTTP status codes 599. Please also add a test for that.
```suggestion
  } else if (statusCode >= 500 && statusCode < 600) {
```

Reply:
gotcha, thanks!